meson.build

```diff
@@ -147,12 +147,13 @@ root_incdir = include_directories('.')
 subdir('libcolord-gtk')
 subdir('po')
 
-if get_option('gtk3')
+with_client = get_option('gtk3')
```

Comment:
why the temp variable?

Reply:
Just so that when we port the client to GTK4, we do not forget to update the check for the `man` pages.

Reply:
Not convinced; I think just check `get_option('gtk3')` twice I think.

Reply:
We could also have a `client` option, side-stepping the need to bind it to a semantically unrelated `gtk3` option.